app/views/questions/upload-supporting-documentation.html

```diff
@@ -7,26 +7,50 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <div class="govuk-caption-m">{{ title }}</div>
-      <h1 class='govuk-heading-l'>Upload supporting documents</h1>
+      <h1 class='govuk-heading-l'>
+        Upload supporting documents
+       </h1>
       <p class="govuk-body">
-        Upload supporting documentation for your {{ title }} form below.
-      </p>
+        Upload extra information or evidence in support of your {{ title }} application.
+       </p>
+
       <div class="govuk-form-group">
         <label class="govuk-label govuk-label govuk-!-font-weight-bold" for="supporting-documentation-upload">
-          Upload supporting documentation
+          Choose supporting documents to upload
         </label>
-        <input class="govuk-file-upload" id="supporting-documentation-upload" name="supporting-documentation-upload" type="file" multiple>
+        <p class="govuk-hint" id="supporting-documentation-upload-hint">Files must be smaller than 10MB and in either a .doc, .gif, .jpg, .pdf, .png, .txt, .xls, or .zip format.</p>
+        <input
+          class="govuk-file-upload"
+          id="supporting-documentation-upload"
+          name="supporting-documentation-upload"
+          aria-describedby="supporting-documentation-upload-hint"
+          type="file"
+          multiple
+        >
       </div>
       <input type="submit" class="govuk-button govuk-button--secondary" name="upload-multiple" value="Upload">
       {% if data['supporting-documentation'] | length > 0 %}
-      <h3 class="govuk-heading-s">Uploaded files</h3>
-      <ul class="govuk-list">
-        {% for file in data['supporting-documentation'] %}
-        <li>{{ file }} – <a class="govuk-link govuk-link--no-visited-state" href="/remove-file?filename={{file}}">Remove<span class="govuk-visually-hidden"> {{ file }}</span></a></li>
+
+        <h3 class="govuk-heading-s">Uploaded files</h3>
+        <ul class="govuk-list">
+          {% for file in data['supporting-documentation'] %}
```

Comment:
There was a missing opening `<li>` tag here that I've added otherwise the list doesn't know how many items it has in it, otherwise I've just updated a few margins for consistenctyy